### PR TITLE
New Plugin: linode-cli

### PR DESCRIPTION
**Pull request by alexclst**
A plugin for the Linode CLI.

Closes https://github.com/1Password/shell-plugins/issues/80

### CODE REVIEW

**Comment by techcraver** [comment]
thank you! This is awesome.

**Review by DCKcode — DISMISSED** [review]
This is great - thanks for the contribution 😄

File: plugins/linode/personal_access_token.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
```
Comment (florisvdg) [inline plugins/linode/personal_access_token.go:33]: Would be nice in the future to also support importing from [`~/.config/linode-cli`](https://www.linode.com/docs/products/tools/cli/guides/install/#interactive-configuration), but we can also do that in a follow-up PR. Created an [issue](https://github.com/1Password/shell-plugins/issues/91) for it.

**Review by florisvdg — COMMENTED** [review]
Great! For CI to pass, could you merge `main` back into your branch?

**Comment by alexclst** [comment]
@florisvdg I believe that I've merged `main` back into my fork, but since this is my first pull request on any open source project not certain. As to the importer, if we are to someday add the config file importer, maybe we should leave it as-is for now to be ready for that?

**Review by SimonBarendse — APPROVED** [review]

**Pull request merged**